From bcd4bd7e0183203aadd37f83c1c623f14276d72f Mon Sep 17 00:00:00 2001
From: Rich Megginson <rmeggins@redhat.com>
Date: Thu, 10 Oct 2013 15:55:29 -0600
Subject: [PATCH 142/225] Ticket #47550 logconv: failed logins: Use of
 uninitialized value in numeric comparison at logconv.pl line 949

https://fedorahosted.org/389/ticket/47550
Reviewed by: nhosoi (Thanks!)
Branch: 389-ds-base-1.3.0
Fix Description: Copy/paste error.  Changed badPassword to badPasswordIp.
Platforms tested: RHEL6 x86_64
Flag Day: no
Doc impact: no
(cherry picked from commit 322fd7996aa810db1b51882d8d1103e11a36cc62)
(cherry picked from commit 86000e43d69267d0b6b8110dcdba43352868c173)
(cherry picked from commit 5844c565267cf23f8a1c4ca5177d26cd82809392)
(cherry picked from commit c5764fb35b1375ceaa2a399794aa5636a00f3b53)
(cherry picked from commit 7369ae484bdeac020cfc113973344052b8636209)
---
 ldap/admin/src/logconv.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/admin/src/logconv.pl b/ldap/admin/src/logconv.pl
index 7381a66..8b423ca 100755
--- a/ldap/admin/src/logconv.pl
+++ b/ldap/admin/src/logconv.pl
@@ -946,7 +946,7 @@ if ($verb eq "yes" || $usage =~ /f/ ){
 			}
 			print "\nFrom the IP address(s) :\n\n";
 			$bpCount = 0;
-			foreach my $ip (sort {$badPassword{$b} <=> $badPassword{$a} } keys %badPasswordIp){
+			foreach my $ip (sort {$badPasswordIp{$b} <=> $badPasswordIp{$a} } keys %badPasswordIp){
 				if ($bpCount > $sizeCount){ last;}
 				$bpCount++;
 				printf "%-4s        %-16s\n", $badPasswordIp{$ip}, $ip;
-- 
1.8.1.4

